cloudsql-postgresql-plugin/src/main/java/io/cdap/plugin/cloudsql/postgres/CloudSQLPostgreSQLUtil.java

```diff
@@ -35,27 +37,45 @@ public class CloudSQLPostgreSQLUtil {
    * @param connectionName Connection Name for the CloudSQL PostgreSQL instance
    */
   public static void checkConnectionName(
-      FailureCollector failureCollector, String instanceType, String connectionName) {
-    
+    FailureCollector failureCollector, @Nullable String instanceType, @Nullable String connectionName) {
+
+    if (Strings.isNullOrEmpty(instanceType)) {
```

Comment:
Same here, and this looks pretty much same as the mysql ones, better combine them

Reply:
ack